services/app-api/src/resolvers/contract/contractResolver.ts

```diff
@@ -44,7 +78,31 @@ export function contractResolver(store: Store): Resolvers['Contract'] {
             }
             return state
         },
-        packageSubmissions(parent) {
+        dateContractDocsExecuted(parent: ParentType) {
+            let dateFirstSubmitted: Date | undefined
+            for (const sub of parent.packageSubmissions) {
+                if (
+                    sub.contractRevision.formData.contractExecutionStatus ===
+                    'EXECUTED'
+                ) {
+                    dateFirstSubmitted =
+                        sub.contractRevision.submitInfo?.updatedAt
```

Comment:
I think this if statement is missing a return here. I'm seeing that a fully executed contract is having this date set to null

<img width="1071" alt="Screenshot 2024-11-26 at 3 13 19 PM" src="https://github.com/user-attachments/assets/3f6aab4a-1026-46ce-b469-7db5859638f0">